koku/masu/util/aws/common.py

```diff
@@ -768,7 +769,7 @@ def delete_s3_objects(request_id, keys_to_delete, context) -> list[str]:
     return []
 
 
-def clear_s3_files(csv_s3_path, provider_uuid, start_date, context, request_id):
+def clear_s3_files(csv_s3_path, provider_uuid, start_date, metadata_key, metadata_value_check, context, request_id):
```

Comment:
Is there currently a case where metadata_key is not "manifestid"? Just wondering if this is worth being a variable if you are just hardcoding it in a different location.

Reply:
So in this function no not right now, but OCP daily is using a different one somewhere else.